lib/cli/run.js

```diff
@@ -286,12 +287,18 @@ exports.builder = yargs =>
       }
 
       // load requires first, because it can impact "plugin" validation
-      handleRequires(argv.require);
+      argv.rawRootHooks = handleRequires(argv.require);
```

Comment:
This should be async as well, in order to remove one of our ESM limitations. Which can't be done in `check()`. Why not use `middleware()` for this job as well?

Reply:
ideally, we'd perform validation during the `check` phase--and it needs `handleRequires` to have been called, first, but yeah, I guess you're right.  `handleRequires` needs to be async anyway.

Reply:
addressed this